stratum/hal/lib/phal/onlp/onlpphal.cc

```diff
@@ -121,10 +125,12 @@ ::util::Status OnlpPhal::Shutdown() {
   absl::WriterMutexLock l(&config_lock_);
 
   // TODO(unknown): add clean up code
+  ::util::Status status;
+  APPEND_STATUS_IF_ERROR(status, phal_db_service_->Teardown());
```

Comment:
Need to move this as per above as well, moving this to the ~AttributeDatabase destructor.